src/bcccp/carpark/paystation/PaystationController.java

```diff
@@ -12,24 +12,44 @@
   private float charge;
 
   public PaystationController(ICarpark carpark, IPaystationUI ui) {
```

Comment:
1. TYPE. Formatting: Google style Infringement
No javadocs present: "At the minimum, Javadoc is present for every public class,
and every public or protected member of such a class".